Framework/LiveData/src/StartLiveData.cpp

```diff
@@ -85,6 +70,38 @@ void StartLiveData::init() {
                   "completes.");
 }
 
+/**
+ * After Instrument property is set copy any properties that the instrument's
```

Comment:
If i'm being picky: I'd say that this comment will not necessarily apply to future modifications since the listener will not be tied to the instrument.